### PR TITLE
Standardisation of the dockerfile and its location

### PR DESCRIPTION
**Pull request by thewhitewizard**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,8 +13,8 @@ RUN groupadd --system appuser \
     && useradd -g appuser -s /sbin/nologin -c "Docker image user" appuser
 
 WORKDIR /app
-COPY $jar app.jar
+COPY $jar blockchain-adapter-api.jar
```
Comment (mcornaton) [inline Dockerfile:16]: We usually use the following format: `iexec-<app-name>.jar` ⬇
```suggestion
COPY $jar iexec-blockchain-adapter-api.jar
```

File: Dockerfile [hunk 2]
```diff
@@ -13,8 +13,8 @@ RUN groupadd --system appuser \
...
 RUN chown -R appuser:appuser /app
 
 USER appuser
-ENTRYPOINT [ "java", "-Djava.security.egd=file:/dev/./urandom", "-jar", "app.jar" ]
+ENTRYPOINT [ "java", "-Djava.security.egd=file:/dev/./urandom", "-jar", "blockchain-adapter-api.jar" ]
```
Comment (mcornaton) [inline Dockerfile:20]:
```suggestion
ENTRYPOINT [ "java", "-Djava.security.egd=file:/dev/./urandom", "-jar", "iexec-blockchain-adapter-api.jar" ]
```

Comment (thewhitewizard) [reply Dockerfile]: it's done

**Review by mcornaton — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -8,46 +8,50 @@ Incoming requests are locally stored in a database.
...
+| `IEXEC_BLOCKCHAIN_ADAPTER_API_WALLET_PATH`              | Path to the wallet of the server.                                                                      | String           | `src/main/resources/wallet.json`             |
+| `IEXEC_BLOCKCHAIN_ADAPTER_API_WALLET_PASSWORD`          | Password to unlock the wallet of the server.                                                           | String           | `whatever`                                   |
 
 ## Health checks
 
-A health endpoint (`/actuator/health`) is enabled by default and can be accessed on the **IEXEC_BLOCKCHAIN_ADAPTER_API_PORT**.
-This endpoint allows to define health checks in an orchestrator or a [compose file](https://github.com/compose-spec/compose-spec/blob/master/spec.md#healthcheck).
-No default strategy has been implemented in the [Dockerfile](docker/Dockerfile) at the moment.
+A health endpoint (`/actuator/health`) is enabled by default and can be accessed on the *
+*IEXEC_BLOCKCHAIN_ADAPTER_API_PORT**.
```
Comment (jbern0rd) [inline README.md:43]: Weird newline here

File: README.md [hunk 4]
```diff
@@ -8,46 +8,50 @@ Incoming requests are locally stored in a database.
...
+No default strategy has been implemented in the [Dockerfile](Dockerfile) at the moment.
 
 ## Running in development mode
 
 `./gradlew docker`
 
 ## CI/CD build
 
-`docker image build -f docker/Dockerfile .`
+`docker image build -f Dockerfile .`
```
Comment (jbern0rd) [inline README.md:54]:
```suggestion
`docker image build .`
```

**Review by jbern0rd — APPROVED** [review]

**Pull request merged**